server/src/main/java/com/linecorp/centraldogma/server/mirror/MirrorSchemes.java

```diff
@@ -50,5 +50,15 @@ public final class MirrorSchemes {
      */
     public static final String SCHEME_GIT_FILE = "git+file";
 
+    /**
+     * {@code "http"}.
+     */
+    public static final String SCHEME_HTTP = "http";
+
+    /**
+     * {@code "http"}.
```

Comment:
I think this annotation has a typo!
```suggestion
     * {@code "https"}.
```